packages/components/src/components/bal-file-upload/bal-file-upload.tsx

```diff
@@ -51,6 +57,16 @@ export class FileUpload {
    */
   @Prop() maxBundleSize: number | undefined = undefined
 
+  /**
+   * Initial files
+   */
+  @Prop() initialFiles: File[] = []
```

Comment:
Instead of initialFiles could we use value and handle it like with an input value, because then we could use it also with two-way binding in angular and vue